lib/pr_checker.js

```diff
@@ -95,8 +95,12 @@ class PRChecker {
       status = false;
       logger.warn(`Approvals: 0`);
     } else {
+      let notComm = approved.length;
+      approved.map((r) => {
+        if (r.review.source !== FROM_REVIEW) { notComm--; }
+      });
       let hint = this.getTSCHint(approved);
-      logger.info(`Approvals: ${approved.length}${hint}`);
+      logger.info(`Approvals: ${notComm}${hint}`);
 
       for (const { reviewer, review } of approved) {
```

Comment:
This can be merged with `review.source === FROM_REVIEW_COMMENT`

Reply:
You mean the two blocks nearly repeating themselves? It probably needs refactoring yes, but I wanted advice on the workflow yet :)